test/e2e_node/dra_test.go

```diff
@@ -39,14 +47,20 @@ const (
 	endpoint               = "/var/lib/kubelet/plugins/test-driver/dra.sock"
 	pluginRegistrationPath = "/var/lib/kubelet/plugins_registry"
 	draAddress             = "/var/lib/kubelet/plugins/test-driver/dra.sock"
+	podTimeout             = time.Second * 30
+	registrationTimeout    = time.Second * 20
```

Comment:
> standard podStartTimeout is 5 minutes. It slowed me down a lot until I realised it. 

That sounds like you were running the test manually, had to wait for the timeout to get the failure message, then try again. That's indeed slow. But it's also slow when using 2 minutes as timeout.

Have you tried running with [`--poll-progress-after=10s`](https://github.com/kubernetes/contributor-site/blob/bb1c87d320aacacc6ec5dd23d312fc59f93e3c6d/content/en/blog/2023/e2e-testing-best-practices.md#tips-for-writing-and-debugging-long-running-tests)?

Then you get feedback after 10 seconds and can decide to wait longer or abort the test. Long timeouts in the test matter a lot less.

> For the simple test pod we're using here 5 minutes is overkill in my opinion.

The same could be said about some simple pods in other E2E tests, and it is indeed very generous. But it's better to be on the safe side and give the CI more time to reduce the amount of test flakes. I'm not convinced that this case here is different.

> I didn't find a plugin registration timeout constant anywhere, will look again.

That indeed is new, so defining it locally is okay. Just add documentation for it so that someone reading the code knows what it's for.

Reply:
Used framework.PodStartShortTimeout constant. It's 2 minutes, which is more or less bearable.

However, I've noticed that using local arbitrary timeout constants or even hardcoded values is not an unusual practice in the current e2e code. 
I'd leave registrationTimeout constant in the code if you don't mind. Or I can add PluginRegistrationTimeout to the framework constants in the test/e2e/framework/util.go if you prefer.

Reply:
> Just add documentation for it so that someone reading the code knows what it's for.

done

> Have you tried running with [--poll-progress-after=10s](https://github.com/kubernetes/contributor-site/blob/bb1c87d320aacacc6ec5dd23d312fc59f93e3c6d/content/en/blog/2023/e2e-testing-best-practices.md#tips-for-writing-and-debugging-long-running-tests)

Thank you. I tried it and it works. A bit more verbose than I expected, but definitely helps. Thank you!